docs/source/rules.rst

```diff
@@ -252,10 +253,16 @@ but those keys can be nested anywhere in the record structure.
 outputs
 ~~~~~~~
 
-Defines the alert destination if the return value of a rule is ``True``.
+``outputs`` defines the alert destination if the return value of a rule is ``True``.
```

Comment:
Please rephrase this (and below) to say "The ``outputs`` keyword argument defines ..."

and "The ``dynamic_outputs`` keyword argument can ...`